index.html

```diff
@@ -26,273 +26,309 @@
     <script src="index.js">
         
     </script>
+    <!--Awesome Fonts!-->
```

Comment:
- [ ] OPTIONAL
Kindly endeavor to remove **all** comments in your code before hosting on the repository, so as to keep your codebase cleaner.